RMS-58423

dandrianov-iiko opened the pull request:
Документация по новым полям АПИ

Создал страницу документации по задаче RMS-58423

dandrianov-iiko committed:
RMS-58423

Документация по новым полям АПИ

Создал страницу документации по задаче RMS-58423

OlgaBelilovskaya commented on _posts/2025-06-02-new-fields-external-address-service.md:
```diff
@@ -0,0 +1,23 @@
+tags: v9preview5 v9
+---
+
+Начиная с V9Preview5 в новом формате адреса появились новые параметры.
```
новый новый - повтор, надо заменить.
Точка перед Title лишняя
новом формате адреса  = адрес в формате Line1
Но - судя по ссылке на IAddress, эти 3 поля просто появились в IAddress, а не в новом формате адреса адреса?

В первом абзаце должно максимально емко и максимально информативно быть описана суть доработки.
Мне абсолютно непонятно из 1го абзаца, что за инфа будет лежать в новых полях (даже distance не понятен, хоть и говорящее название казалось бы)

Надо в 1м абзаце указать, что за информация появилась в IAdress, можно без ссылок даже,  это не обязательно, просто по-русски, чтобы было понятно, что за новая инфа будет храниться в IAdress

И главное, непонятно, для чего эти новые поля добавились. Фраза " более информативный вывод по сравнению с тем, что ранее выводилось только ['Line1']" - ну ээ, что за вывод, куда вывод.
Появились новые поля такие то, в низ будет храниться инфа такаято (если это возможно описать словами). Этим можно пользоваться будет при ....

dandrianov-iiko replied:
Постарался дополнить полезной информацией.

OlgaBelilovskaya commented on _posts/2025-06-02-new-fields-external-address-service.md:
```diff
@@ -0,0 +1,23 @@
+---
+
+Начиная с V9Preview5 в новом формате адреса появились новые параметры.
+['Title'](https://iiko.github.io/front.api.sdk/v9/html/P_Resto_Front_Api_Data_Brd_IAddress_Title.htm), ['Subtitle'](https://iiko.github.io/front.api.sdk/v9/html/P_Resto_Front_Api_Data_Brd_IAddress_Subtitle.htm) и ['Distance'](https://iiko.github.io/front.api.sdk/v9/html/P_Resto_Front_Api_Data_Brd_IAddress_Distance.htm).
```
тут одинарные кавычки вокруг названий полей не нужны. Вместо них должны стоять ` вокруг названий классов и свойств и тп, что переводит обычный шрифт в моноспейс или как там. Посмотри как в других заметках сделана разметка такая.

dandrianov-iiko replied:
С кавычками конкретно попутал. Исправил!

OlgaBelilovskaya commented on _posts/2025-06-02-new-fields-external-address-service.md:
```diff
@@ -0,0 +1,23 @@
+Отправной точкой считается адрес торгового предприятия.
+
+
+Примеры создания адреса можно найти в проекте SDK SamplePlugin в классе EditorTester.cs: CreateDelivery и ChangeDeliveryOrderTypeOnCourier.
```
названия нужно в ` написать имхо, чтобы шрифт отличался.

в классе EditorTester.cs: **в методах** CreateDelivery и ChangeDeliveryOrderTypeOnCourier.

Посмотри статью про Семпл плагин
https://iiko.github.io/front.api.doc/2024/04/12/sampleplugin-update.html
оформи тут ссылки и на сам плагин и на методы аналогично, пжл.

OlgaBelilovskaya commented on _posts/2025-06-02-new-fields-external-address-service.md:
```diff
@@ -0,0 +1,23 @@
+Эти поля нужны для облегчения поиска нужного адреса, более информативный вывод по сравнению с тем, что ранее выводилось только ['Line1'](https://iiko.github.io/front.api.sdk/v9/html/P_Resto_Front_Api_Data_Brd_IAddress_Line1.htm).
+
+Для переключения на новые адреса и дальнейшее использование преимуществ новых полей необходимо убедиться в следующем:
+- в настройках бэка выбран новый формат адреса (администрирование-настройки торгового предприятия-формат адреса доставки-использовать новый формат адреса)
```
бэка -> BackOffice
(администрирование-настройки торгового предприятия-формат адреса доставки-использовать новый формат адреса) -> 
(Администрирование -> Настройки торгового предприятия -> Формат адреса доставки -> Использовать новый формат адреса)

OlgaBelilovskaya commented on _posts/2025-06-02-new-fields-external-address-service.md:
Основное, что нужно - попробовать перефразировать Очень сумбурно написано и что сделано и для чего. Перечислены действия, которые необходимо осуществить, но плохо понятно для чего.
Если надо, можно подумать вместе.

dandrianov-iiko committed:
RMS-58423

Документация по новым полям АПИ

Отредактировал страницу документации по задаче RMS-58423

dandrianov-iiko committed:
RMS-58423

Документация по новым полям АПИ

Исправление замечаний после ревью

dandrianov-iiko committed:
RMS-58423

Документация по новым полям АПИ

Исправление замечаний после ревью - 2
